projects/addon-doc/src/components/navigation/navigation.style.less

```diff
@@ -55,7 +55,7 @@
     .clearbtn();
     font: var(--tui-font-text-m);
     margin-left: 0.5rem;
-    line-height: 2.75rem;
+    padding-top: 0.6rem;
```

Comment:
This is not enough. There's also `&_small` below and even if I add the same fix, scaling it according to smaller line-height I get this:
![image](https://user-images.githubusercontent.com/11832552/145975789-4fc311c1-aabc-4409-92db-63484426653f.png)

Please try multiline in all possible situations for the navigation menu:
![image](https://user-images.githubusercontent.com/11832552/145975966-c0ae6b16-e753-4efd-a282-5ed624ad84be.png)

Reply:
I should decrease the line-height on each of these menu items or I'm missing something?

I don't quite understand how the result should look like

Reply:
For example, for the **Common** accordion item, we will have to remove the text-overflow styles because it won't allow doing multiline
```
.title {
    .text-overflow();
    margin-right: 0;
    flex-grow: 1;
}
```

Reply:
This is how it looks like now
![image](https://user-images.githubusercontent.com/54478076/147274983-bcc4ab50-6288-453d-a950-fbda314ef255.png)

Reply:
Let's make it 1rem for `sublink` and 0.5rem for `sublink_small`. And we also need to get rid of this rule:
```css
        &:not(:first-child) {
            margin-top: 1rem;
        }
```
And then I believe we're good to merge

Reply:
To which property apply 1rem and 0.5rem?
0.5rem line-height not gonna work for `sublink_small`
![image](https://user-images.githubusercontent.com/54478076/147286571-b363b744-1338-4181-9013-86b2a1d62fbd.png)

Reply:
Yeah, I meant padding, sorry